packages/react-server-cli/src/commands/init.js

```diff
@@ -0,0 +1,71 @@
+import _ from "lodash";
+import fs from "fs";
+import {spawnSync} from "child_process";
```

Comment:
There's no concurrency among subprocesses here.  Easier flow of control this way.